spec.md

```diff
@@ -224,7 +224,10 @@ help intermediate gateways determine how to route the events.
 * Constraints:
   * REQUIRED
   * MUST be a non-empty string
-  * MUST be unique within the scope of the producer
+  * MUST be unique within the scope of the producer. In other words:
+    events with identical "scope" and "id" values MUST be duplicates
+    that can be discarded without examining any other attribute
+    values.
```

Comment:
The MUST reads strange from a consumer perspective. As a consumer, I can treat them as duplicates, but there is no guarantee they really are. Counter-proposal (including what I've said above):

> In other words: events with identical `id`, `source` and `type` values CAN be discarded without examining any other attribute values.